properties.py

```diff
@@ -203,7 +203,7 @@ def update_name(self, context):
 
 aov_mapping = [ 
                  
-                 ("rgba", "Combined (rgba)", "Combined (rgba)"),
+                 ("Ci,a", "Combined (rgba)", "Combined (rgba)"),
```

Comment:
I used that as a shortcut for multilayer export.  Reviewing my code 'rgba' would work just as well.